pom.xml

```diff
@@ -92,17 +92,50 @@
 
         <includedTestGroups />
         <excludedTestGroups>Integration,Live</excludedTestGroups>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        
+        <!-- Dependencies -->
         <guava.version>18.0</guava.version>
         <javatuples.version>1.2</javatuples.version>
         <brooklyn.version>0.7.0-20150614.2158</brooklyn.version>
         <jetty.version>9.2.4.v20141103</jetty.version>
         <dropwizard.version>0.8.1</dropwizard.version>
+        <httpcomponents.version>4.3.6</httpcomponents.version>
+        <gson.version>2.3</gson.version>
+        <json.version>20140107</json.version>
+        <tomcat-servlet.version>8.0.21</tomcat-servlet.version>
+        <json-simple.version>1.1.1</json-simple.version>
+        <alien4cloud.version>1.0.0-SM26</alien4cloud.version>
+        <commons-io.version>2.4</commons-io.version>
+        <spring.version>4.1.4.RELEASE</spring.version>
+        <spring-security.version>4.0.1.RELEASE</spring-security.version>
+        <commons-lang.version>3.3.2</commons-lang.version>
+        <validation-api.version>1.1.0.Final</validation-api.version>
+        <jersey.version>1.18.1</jersey.version>
+        <jackson.version>2.4.5</jackson.version>
+        <hibernate.version>4.2.7.Final</hibernate.version>
+        <hibernate-jpa.version>1.0.0.Final</hibernate-jpa.version>
+        <hibernate-tools.version>2.1.3</hibernate-tools.version>
+        <commons-httpclient.version>3.1</commons-httpclient.version>
+        <toscamodel.version>1.0.0</toscamodel.version>
+        <hsqldb.version>2.3.0</hsqldb.version>
+        <mysql-connector.version>5.1.27</mysql-connector.version>
+        <modaclouds.version>2.4</modaclouds.version>
+        <jaxb.version>2.2.6</jaxb.version>
+        <castor.version>1.2</castor.version>
+        
+        <!-- Test dependencies -->
         <junit.version>4.11</junit.version>
```

Comment:
I think we should take the change to choose between junit or testng as having both already created some headaches.
I vote for testng as it supports groups more easily than junit, but not strong feelings.

No need to address this comment in this PR